src/zope/app/apidoc/utilities.rst

```diff
@@ -166,13 +166,15 @@ For interfaces we simply get
   >>> utilities.getPythonPath(ISample)
   'zope.app.apidoc.doctest.ISample'
 
-and for classes
+and for classes (note that we must pass a "bound method", because the
+unbound method is no longer present in Python 2):
 
   >>> class Sample(object):
   ...     def sample(self):
   ...         pass
 
-  >>> utilities.getPythonPath(Sample.sample)
+  >>> PY3 = str is not bytes
+  >>> utilities.getPythonPath(Sample().sample if PY3 else Sample.sample)
   'zope.app.apidoc.doctest.Sample'
```

Comment:
(It's getting later on this side of the world, and I can feel the clarity of my writing waning fast.)

I mean Python 3 should produce the same output as Python 2 currently does when passed `Sample.sample`, and `__qualname__` is the only way I know how to do that on Python 3.

I love `__qualname__` BTW.